packages/filebrowser-extension/src/index.ts

```diff
@@ -1157,48 +1163,71 @@ function addCommands(
   });
 
   app.contextMenu.addItem({
-    command: CommandIDs.rename,
+    type: 'separator',
     selector: selectorItem,
     rank: 4
   });
+
   app.contextMenu.addItem({
-    command: CommandIDs.del,
+    command: CommandIDs.rename,
     selector: selectorItem,
     rank: 5
   });
+
   app.contextMenu.addItem({
-    command: CommandIDs.cut,
+    command: CommandIDs.del,
     selector: selectorItem,
     rank: 6
   });
 
+  app.contextMenu.addItem({
+    command: CommandIDs.cut,
+    selector: selectorItem,
+    rank: 7
+  });
+
   app.contextMenu.addItem({
     command: CommandIDs.copy,
     selector: selectorNotDir,
-    rank: 7
+    rank: 8
   });
 
   app.contextMenu.addItem({
     command: CommandIDs.duplicate,
     selector: selectorNotDir,
-    rank: 8
+    rank: 9
+  });
+
+  app.contextMenu.addItem({
+    type: 'separator',
+    selector: selectorItem,
+    rank: 10
   });
+
   app.contextMenu.addItem({
     command: CommandIDs.shutdown,
     selector: selectorNotDir,
-    rank: 10
+    rank: 11
+  });
+
+  app.contextMenu.addItem({
+    type: 'separator',
+    selector: selectorItem,
+    rank: 12
   });
 
   app.contextMenu.addItem({
     command: CommandIDs.share,
```

Comment:
One minor nit, since it does bug me a little: could you please alphabetize the commands in the "copy special" subsection?

So the ranks would look like:

- `copyFromPath` -> `rank: 13`
- `copyPath` -> `rank: 14`
- `share` -> `rank: 15`

The code adding the `copyFromPath` menu item is lower down in the file in a separate plugin: https://github.com/jupyterlab/jupyterlab/blob/7589731dc699d53830aadd89ff622614b0bfcec8/packages/filebrowser-extension/src/index.ts#L372-L376.

This does bug me a bit, but I can always just take care of it myself later in a new PR

Reply:
Thanks for the review! I've made the changes.

I've got a small concern regarding backward compatibility, I guess renaming a command is considered backward incompatible?

Reply:
> renaming a command is considered backward incompatible?

Good point. One way to deal with that would be to re-add the `filebrowser:share` command as an alias of the `filebrowser:copyShareableLink`, and then remove the `filebrowser:share` alias when we release jlab 4.0.

Let me see if there's an actual reasonable way to implement such an alias. Otherwise we can always revert the change to `copyShareableLink`